### ISSUE TITLE
Just notifying about a similar project

### ISSUE THREAD

**Issue by luxas** [runseb/k8s4pi#1]
Hello,

This isn't a "real" issue but it was the first way I thought of in order to contact you.

I've a similar project, but that doesn't depend on `andrewpsuedonym`.
It has the name [kubernetes-on-arm](https://github.com/luxas/kubernetes-on-arm)

All source code how to build the latest versions is there and even how to make the sd card.

It would be great if you could check it out and eventually give some suggestions
After all, we're trying to do the same thing :smile:

**Comment by sebgoa** [comment]
cool @luxas I will take a look , we are definitely doing the same thing :)

**Comment by luxas** [comment]
WDYT? I see this has been renamed to skippbox

**Comment by sebgoa** [comment]
@luxas, yes I moved all my Kubernetes related project under one organization 'skippbox'.
